discovery/pkg/apigee/pollproxiesjob_test.go

```diff
@@ -260,10 +274,25 @@ func (m mockProxyClient) GetRevisionPolicyByName(apiName, revision, policyName s
 
 func (m mockProxyClient) IsReady() bool { return false }
 
-type mockProxyCache struct{}
+type mockProxyCache struct {
+	pathSpec bool
+	nameSpec bool
+}
 
 func (m mockProxyCache) GetSpecWithPath(path string) (*specCacheItem, error) {
-	return &specCacheItem{}, nil
+	if m.pathSpec {
+		return &specCacheItem{}, nil
+	} else {
+		return nil, nil
+	}
+}
+
+func (m mockProxyCache) GetSpecWithName(path string) (*specCacheItem, error) {
```

Comment:
lol, i know this func and GetSpecWithPath are mock/test stuff, but path string is not being used :)